index.html

```diff
@@ -67,65 +81,88 @@ <h3 class="headthree">I am a Software Developer</h3>
     </ul>
     <br>
 </div>
-                                    <!--My works-->
+
+
+                                    <!--My works-->                         
 <section class="mainmyworks">
         <div class="worksheading">
-            <div>
+            <div class="workhead">
         <h2 class="headtwo">My Recent works</h2>
             </div>
             <div>
                 <hr class="liner">
             </div>
-        </div>
         <div class="workitems">
                     <div class="projectno1">
                         <div class="photo">
```

Comment:
The background color is white, so please make sure to replace the light gray with white.

You have a lot of space from the top in your work section and the content in your featured project needs some arrangement. So please make sure to update the layout a little bit for this section.

![image](https://user-images.githubusercontent.com/34779928/194695427-e0604d0d-e827-4b5b-8204-45739f68543e.png)